src/modules/api/user/user.dto.ts

```diff
@@ -0,0 +1,18 @@
+import { IsNotEmpty, IsString, IsDate } from 'class-validator';
+export class UserDTO {
+  @IsNotEmpty()
+  @IsString()
+  _id: string;
+
+  @IsString()
+  name: string;
+
+  @IsDate()
+  joinedAt: Date;
+
+  @IsString()
+  emailAddress: string;
+
+  @IsString()
+  phoneNumber: string;
+}
```

Comment:
Please get any one of your PR first. All three of your PRs are showing these changes.